cfme/tests/networks/test_sdn_subnets.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from cfme import test_requirements
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.networks.views import SubnetView
+
+pytestmark = [
+    test_requirements.sdn,
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.provider([OpenStackProvider], scope="module")
+]
+
+
+@pytest.fixture()
+def network_manager(appliance, provider):
```

Comment:
Well, in the future when they add another providers with network managers the fixture makes sense.

Reply:
@jaryn valid concern for sure.

I wouldn't consider this a blocker to merging this PR, but warrants some future enhancement.

This could be a globally available fixture (cfme/fixtures/), or even a subcollection of the parent cloud provider.